src/main/test/kotlin/fr/yvbro/rexa/UserTest.kt

```diff
@@ -0,0 +1,59 @@
+package fr.yvbro.rexa
+
+import fr.yvbro.rexa.exception.RexaAuthentificationFailedException
+import fr.yvbro.rexa.model.User
+import fr.yvbro.rexa.repository.UserRepository
+import fr.yvbro.rexa.service.UserService
+import org.junit.jupiter.api.*
+import org.mindrot.jbcrypt.BCrypt
+import org.mockito.InjectMocks
+import org.mockito.Mock
+import org.mockito.Mockito.*
+import org.springframework.boot.test.context.SpringBootTest
+
+
+@Tag("unitTest")
+@DisplayName("class User")
+@SpringBootTest
+class UserTest {
+
+
+    @Nested
+    inner class UserServiceTest {
+
+        @InjectMocks
+        lateinit var userService: UserService
+
+        @Mock
+        lateinit var userRepository: UserRepository
+
+        @BeforeEach
+        fun setMock() {
+            val bCrypt = mock(BCrypt::class.java)
+            var goodUserFromRepository = User("beber@gmail.com", "\$2a\$04\$/FCeArg6BcWT7tYqChau5uhLlVaxOa4c6v5gO19mTAEW/4BPMXWiy")
+
+            `when`(userRepository.getUserByEmail(any())).thenReturn(goodUserFromRepository)
+        }
+
+        @Test
+        fun `If we have the same password, we don't throw exception`() {
```

Comment:
They recommande to name the method like this ? Because there is an extra annotations @Display for the method where you can give a string like this.